### DISCUSSION TITLE
Looking for help with issues related to LDAP server binding authentication

### DISCUSSION THREAD

**Discussion by 136750162** [WrenSecurity/wrends#67] [Q&A]
At present, I want to implement the binding operation request from the client to verify whether the user name exists. If it exists, it will forward the request to the relevant third-party password authentication platform for authentication. If it does not exist, it will directly return the response of authentication failure.

**Comment by pavelhoral** [comment]
Just want to let you know I want to get to this question when I have time. I am currently preoccupied with other high priority work.
